demos/profiling/src/utils.py

```diff
@@ -147,54 +166,6 @@ def get_model(
     return model_trt
 
 
-def get_euclidean_distance(video_width, video_height):
-    def euclidean_distance(detected_pose, tracked_pose):
-        difference = detected_pose.points - tracked_pose.estimate
-        difference[:, 0] /= video_width
-        difference[:, 1] /= video_height
-        distances = np.linalg.norm(difference, axis=1)
-
-        track_points_to_use = tracked_pose.last_detection.scores > DETECTION_THRESHOLD
-        detections_to_use = detected_pose.scores > DETECTION_THRESHOLD
-
-        indices_to_use = np.argwhere(
-            np.logical_and(track_points_to_use, detections_to_use)
-        )
-
-        if len(indices_to_use) > 0:
-            return np.mean(distances[indices_to_use])
-        else:
-            return DISTANCE_THRESHOLD + 1
-
-    return euclidean_distance
-
-
-def keypoints_distance(detected_pose, tracked_pose):
-    distances = np.linalg.norm(detected_pose.points - tracked_pose.estimate, axis=1)
-
-    match_num = np.count_nonzero(
-        (
-            distances
-            < KEYPOINT_DIST_SCALE_FACTOR
-            * np.linalg.norm(tracked_pose.estimate[11] - tracked_pose.estimate[15])
-        )
-        * (detected_pose.scores > DETECTION_THRESHOLD)
-        * (tracked_pose.last_detection.scores > DETECTION_THRESHOLD)
-    )
-    return 1 / (1 + match_num)
```

Comment:
As mentioned on a previous comment, maybe we can leave this distance function in here only for this demo. Or make a similar one `rescaled_keypoints_vote` which considers the size of the objects (use something like the diameter, make it generic in that case and not subject to the detector as in this definition)